### PR TITLE
run both net46 and netcoreapp2.0 FCS tests in CI

### PR DESCRIPTION
**Pull request by dsyme**
I don't understand why, but only the `netcoreapp2.0` run of the tests is appearing in the output logs of `release_fcs_windows_nt_prtest`

The line

        runDotnet __SOURCE_DIRECTORY__ "test FSharp.Compiler.Service.Tests/FSharp.Compiler.Service.Tests.fsproj -v n -c Release"

is equivalent to 

        dotnet test FSharp.Compiler.Service.Tests/FSharp.Compiler.Service.Tests.fsproj -v n -c Release

and for some reason it doesn't run both `net46` and `netcoreapp2.0` tests, e.g. [this log](https://ci2.dot.net/job/Microsoft_visualfsharp/job/master/job/release_fcs_windows_nt_prtest/2394/consoleText).  It's odd because both kinds of tests get run in the same CI running in AppVeyor.  

Anyway this PR tries to force both sets of tests to run

### CODE REVIEW

**Comment by dsyme** [comment]
@dotnet-bot test Ubuntu16.04 Release_fcs Build please

**Comment by dsyme** [comment]
OK we are getting this error:
```
Starting test execution, please wait...
NUnit Adapter 3.9.0.0: Test execution started
Running all tests in D:\j\w\release_fcs_w---49a16363\fcs\FSharp.Compiler.Service.Tests\bin\Release\net46\FSharp.Compiler.Service.Tests.dll
NUnit failed to load D:\j\w\release_fcs_w---49a16363\fcs\FSharp.Compiler.Service.Tests\bin\Release\net46\FSharp.Compiler.Service.Tests.dll
NUnit Adapter 3.9.0.0: Test execution complete
No test is available in D:\j\w\release_fcs_w---49a16363\fcs\FSharp.Compiler.Service.Tests\bin\Release\net46\FSharp.Compiler.Service.Tests.dll. Make sure that test discoverer & executors are registered and platform & framework version settings are appropriate and try again.

```

File: fcs/build.fsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ let isMono = false
 // Utilities
 // --------------------------------------------------------------------------------------
 
-let dotnetExePath = DotNetCli.InstallDotNetSDK "2.1.201"
+let dotnetExePath = DotNetCli.InstallDotNetSDK "2.1.300"
```
Comment (KevinRansom) [inline fcs/build.fsx]: @dsyme 
the DotnetCLIToolsVersion.txt has the version number of the dotnet cli to use.  Can you get the version by reading the version from this file?

https://github.com/Microsoft/visualfsharp/blob/master/DotnetCLIToolsVersion.txt

Comment (dsyme) [reply fcs/build.fsx]: thanks, now fixed

**Comment by dsyme** [comment]
@KevinRansom We're now getting failures due to the technique FCS testing uses to install the .NET SDK.  I guess we need to adjust a bit more

**Pull request closed without merging**